lib/auth/grpcserver_test.go

```diff
@@ -5237,6 +5239,32 @@ func TestGetAccessGraphConfig(t *testing.T) {
 	}
 }
 
+func TestGetVnetConfig(t *testing.T) {
+	server := newTestTLSServer(t)
+	user, _, err := CreateUserAndRole(server.Auth(), "test", []string{"role"}, nil)
+	require.NoError(t, err)
+
+	// Create newConfig.
+	newConfig, err := vnet.NewVnetConfig(&vnetv1pb.VnetConfigSpec{
+		Ipv4CidrRange:  vnet.DefaultIPv4CIDRRange,
+		CustomDnsZones: []*vnetv1pb.CustomDNSZone{&vnetv1pb.CustomDNSZone{Suffix: "example.com"}},
+	})
+	require.NoError(t, err)
+	createdConfig, err := server.Auth().CreateVnetConfig(t.Context(), newConfig)
+	require.NoError(t, err)
+
+	// Verify that a regular user is able to fetch the config.
+	identity := authz.LocalUser{
+		Username: user.GetName(),
```

Comment:
After writing this I realized that there's already [`lib/services/local/vnet_config_test.go`](https://github.com/gravitational/teleport/blob/f930a3dd5ad9c282112ffa09e361fb89eb432e0c/lib/services/local/vnet_config_test.go), though that test doesn't verify if a regular user can fetch the VNet config so I guess it might be worth it to keep it?